### PR TITLE
feedback entity

### PR DESCRIPTION
**Pull request by OlhaLozinska**

### CODE REVIEW

File: src/main/java/com/softserve/academy/studhub/entity/FeedBack.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    @Column(name = "id")
+    private Integer id;
+
+    @EqualsAndHashCode.Exclude
+    @Column(name = "body")
+    private String body;
+
+    @EqualsAndHashCode.Exclude
+    @Column(name = "mark")
+    private int mark;
```
Comment (DaNkOLULzz) [inline src/main/java/com/softserve/academy/studhub/entity/FeedBack.java]: Make mark Integer

Comment (OlhaLozinska) [reply src/main/java/com/softserve/academy/studhub/entity/FeedBack.java]: done

File: src/main/java/com/softserve/academy/studhub/entity/FeedBack.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+@Table(name = "feedback")
+public class FeedBack {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Integer id;
+
+    @EqualsAndHashCode.Exclude
+    @Column(columnDefinition = "TEXT", name = "body")
```
Comment (Jarvizzik) [inline src/main/java/com/softserve/academy/studhub/entity/FeedBack.java]: swap places of the columnDefinition and name parameter(for better readability)

File: src/main/java/com/softserve/academy/studhub/entity/FeedBack.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@EqualsAndHashCode
+@Table(name = "feedback")
+public class FeedBack {
+
```
Comment (volodymyroseredchuk) [inline src/main/java/com/softserve/academy/studhub/entity/FeedBack.java]: Please change the class name to 'Feedback' and table name to 'feedbacks'

Comment (OlhaLozinska) [reply src/main/java/com/softserve/academy/studhub/entity/FeedBack.java]: Done)

**Pull request merged**